src/niMidi.cpp

```diff
@@ -70,6 +105,17 @@ class NiMidiSurface: public BaseSurface {
 	virtual void SetTrackListChange() override {
 	}
 
+	virtual void SetSurfaceVolume(MediaTrack *trackid, double volume) override {
```

Comment:
Note that these were directly copied from reaper_plugin.h

Reply:
> Note that these were directly copied from reaper_plugin.h

Yeah, I suspected as much. I tend to change them, both to fit with the style elsewhere in the code and because "trackid" for a track pointer (as opposed to a numeric identifier) seems kinda weird to me.